internal/mapping_test.go

```diff
@@ -119,3 +119,30 @@ func TestGetSpannerKeyName(t *testing.T) {
 		assert.Equal(t, tc.spKeyName, spKeyName, tc.name)
 	}
 }
+
+func TestGetSpannerIndexKeyName(t *testing.T) {
```

Comment:
Instead of duplicating these tests, consider testing the util function `func getUniqueId(srcId string, spannerIds map[string]bool) string` itself. It will make it far easier to update tests in case this function changes in the future.

Reply:
Actually ToSpannerIndexKey and ToSpannerForeignKey behaves differently when source key name is empty string, considering that we need to test these functions separately

Reply:
updated this, now testing getUniqueId and empty string case separately